packages/flutter/lib/src/foundation/isolates.dart

```diff
@@ -44,4 +43,7 @@ typedef ComputeImpl = Future<R> Function<Q, R>(ComputeCallback<Q, R> callback, Q
 ///
 /// The `debugLabel` argument can be specified to provide a name to add to the
 /// [Timeline]. This is useful when profiling an application.
+typedef ComputeImpl = Future<R> Function<Q, R>(ComputeCallback<Q, R> callback, Q message, { String? debugLabel });
+
+/// See [ComputeImpl].
```

Comment:
This doc on `compute` is not very informative. Maybe we could use macros to just repeat the docs on both?

Reply:
I'll change it to briefly describe what `compute` does, but I think that documenting the parameters here is worse because readers can't see what the parameters actually are, and we should explicitly direct readers to the `typedef`.

Reply:
This should say why I should look there (e.g. for a documentation of the parameters and return value of the function).

Reply:
Fair point.  I've changed it to "... for usage details." for brevity.  Do you think that's sufficient?